src/test/HostActivationTests/GivenThatICareAboutSharedFxLookup.cs

```diff
@@ -358,20 +358,6 @@ public void Roll_Forward_On_No_Candidate_Fx_Minor_And_Disabled()
             // Add some dummy versions in the exe
             SharedFramework.AddAvailableSharedFxVersions(_builtSharedFxDir, _exeSharedFxBaseDir, "10000.1.1");
 
-            string expectedPrereqInstallUrl;
-            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
```

Comment:
These were the only tests for the platform specific links.
Can you please move this to a test which exercises the other cases where the links are used?